packages/styles/src/components/header/index.scss

```diff
@@ -4,6 +4,28 @@
 @use 'mixins' as *;
 
 post-header {
+  --global-header-height: 72px;
+  --global-header-minimal-height: 24px;
```

Comment:
Don't know if this one is used, I think this was renamed to `--global-header-reduced-height`. Also, it should probably be set to `--global-header-reduced-height: --global-header-height;` on mobile... 
Can you please check this out and clean the variables?